dart/looker_sdk/lib/src/rtl/auth_token.dart

```diff
@@ -0,0 +1,42 @@
+class AuthAccessToken {
+  final String _accessToken;
+  final String _tokenType;
+  final int _expiresIn;
+
+  AuthAccessToken.fromResponse(Map map, String contentType)
+      : _accessToken = map['access_token'],
+        _tokenType = map['token_type'],
+        _expiresIn = map['expires_in'];
```

Comment:
Do you think you'll want the refresh token in case someone wants to implement OAuth flow? Or worry about it then?